language/German/strings.po

```diff
@@ -884,21 +885,26 @@ msgstr "WAN"
 
 #: src/frontend/mame/ui/info.cpp:70
 msgid ""
-"One or more ROMs/disk images for this system are incorrect. The system may "
-"not run correctly.\n"
+"One or more ROMs/disk images for this system are incorrect. The system may not "
+"run correctly.\n"
```

Comment:
Please back out this change rewrapping the English message.

Reply:
Fixed